hardhat.config.ts

```diff
@@ -162,6 +175,7 @@ const config: HardhatUserConfig = {
         : { mnemonic: MNEMONIC, accountsBalance: (10n ** 36n).toString() },
       // this should only be relied upon for test harnesses and coverage (which does not use viaIR flag)
       allowUnlimitedContractSize: true,
+      hardfork: "shanghai"
```

Comment:
Writing this down for posterity:

So after rebasing, I started getting the following error when spidering Goerli:

```
A withdrawalsRoot for a header can only be provided with EIP4895 being activated
```

I believe this issue was caused by upgrading to the latest hardhat version ([2.13.1](https://github.com/NomicFoundation/hardhat/releases/tag/hardhat%402.13.1)), which upgraded the `ethereumjs` dependency version.

That new version of ethereumjs included an upgrade to support EIP-4895 (staking withdrawals for the Shanghai upgrade): https://github.com/ethereumjs/ethereumjs-monorepo/pull/2353

Those changes included new checks that a `withdrawalsRoot` header is only provided when on a chain with 4895 activated.

Goerli has already undergone the Shanghai upgrade, and so it was failing these new checks. Specifying that our local hardhat network should be set to Shanghai (and therefore enable 4895) fixes the issue.